tests/baselines/reference/user/webpack.log

```diff
@@ -0,0 +1,20 @@
+Exit Code: 2
+Standard output:
+lib/dependencies/WorkerPlugin.js(268,23): error TS2339: Property 'name' does not exist on type 'Record<string, any> | {}'.
```

Comment:
This variable flows in from a location pretty explicitly typed as `Record<string, any>` - the `| {}` seems pretty bogus; not sure how it got in there, but it definitely seems like a bug.